smt/applications/tests/test_ego.py

```diff
@@ -230,16 +228,16 @@ def test_branin_2D_mixed(self):
 
     @staticmethod
     def function_test_cate_mixed(X):
-        x1 = X[:, 0].astype(float)
+        x1 = X[:, 0]
         #  cate 1
         c1 = X[:, 1]
-        x2 = c1 == "1"
```

Comment:
In real test cases, users defined categorical variables with names (or labels) which are strings.

It would be great if the users can gives the labels and the the toolbox gives back the point corresponding to a real label.

Il my paint can be "blue" or "green", I want my model outputs to be "blue" paint or "green" paint.